sections/gcloud.zsh

```diff
@@ -31,8 +31,15 @@ spaceship_gcloud() {
   # Check if there is an active config
   [[ -f ${GCLOUD_DIR}/active_config ]] || return
 
-  # Reads the current config from the file
-  local GCLOUD_ACTIVE_CONFIG=$(head -n1 ${GCLOUD_DIR}/active_config)
+  # Check if there is an active config override
+  if (( ${+CLOUDSDK_ACTIVE_CONFIG_NAME} ))
+  then
```

Comment:
```suggestion
  if (( ${+CLOUDSDK_ACTIVE_CONFIG_NAME} )); then
```

Reply:
We use single line conditions

Reply:
The pull request has been updated to include the suggested change.